### ISSUE TITLE
Palbrough/Giddeus/Ghelsba/Fort Ghelsba/Yughott Grotto repop timers incorrect

### ISSUE THREAD

**Issue by ShiyoKozuki** [EdenServer/community#2368]
### Checklist

- [x] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [x] I have provided reproducable steps. (No "as the title says" posts please. Provide reproducable steps even if it seems like a waste of time.)
- [x] I have provided my client version in the details. (type /ver into your game window)

The repop timers for all mobs in all of these zones are too quick. Here are the correct repop timers.

Giddeus(Outdoors)

https://ffxiclopedia.fandom.com/wiki/Yagudo_Piper  8 minutes
https://ffxiclopedia.fandom.com/wiki/Yagudo_Persecutor 8 minutes
https://ffxiclopedia.fandom.com/wiki/Yagudo_Mendicant 8 minutes
https://ffxiclopedia.fandom.com/wiki/Yagudo_Acolyte 8 minutes
https://ffxiclopedia.fandom.com/wiki/Yagudo_Initiate 8 minutes
Level 3-18 mobs are supposed to be 8 minutes.
https://ffxiclopedia.fandom.com/wiki/Yagudo_Priest
https://ffxiclopedia.fandom.com/wiki/Yagudo_Theologist
https://ffxiclopedia.fandom.com/wiki/Yagudo_Votary
Level 21-23 mobs are supposed to be 10 minutes.

Palborough MInes(Indoors)

https://ffxiclopedia.fandom.com/wiki/Amber_Quadav 16 minutes
https://ffxiclopedia.fandom.com/wiki/Amethyst_Quadav 16 minutes
https://ffxiclopedia.fandom.com/wiki/Young_Quadav 16 minutes
https://ffxiclopedia.fandom.com/wiki/Veteran_Quadav 8 minutes (Inaccurate?)
https://ffxiclopedia.fandom.com/wiki/Onyx_Quadav 16 minutes
https://ffxiclopedia.fandom.com/wiki/Greater_Quadav 16 minutes

https://ffxiclopedia.fandom.com/wiki/Old_Quadav 10 minutes
https://ffxiclopedia.fandom.com/wiki/Copper_Quadav 10 minutes
https://ffxiclopedia.fandom.com/wiki/Brass_Quadav 10 minutes
Level 21-23 mobs are supposed to be 10 minutes.

Ghelsba Outpost(Outdoors)

No info here on the wiki, but if we go by Giddeus it's probably going to be the same pattern. 3-18 mobs 8 minutes.

Fort Ghelsba(Outdoors)
https://ffxiclopedia.fandom.com/wiki/Orcish_Serjeant 10 minutes
https://ffxiclopedia.fandom.com/wiki/Orcish_Fighter 10 minutes
https://ffxiclopedia.fandom.com/wiki/Orcish_Cursemaker 10 minutes
Level 21-23 monsters are supposed to be 10 minutes.

Yughott Grotto(Indoors) 

No info here either, but if we copy Palborough mines it's probably accurate. Level 3-18 monsters are 16 minute repop and 21-23 are 10 minutes.

**Comment by Spiffly** [comment]
To add to this as it still appears to be an issue.

I killed a warrior AF NM in Palborough mines earlier. I cleared away 2 quadav prior to popping the NM. The mobs respawned after approximately 5 minutes and i believe with the mines being a dungeon, the respawn should be around 16 minutes. The mobs can be found at G-10 on floor 3: https://classicffxi.fandom.com/wiki/Palborough_Mines?file=Palborough3.png
